bolt/src/MCPlusBuilder.h

```diff
@@ -1348,6 +1350,13 @@ class MCPlusBuilder {
     return false;
   }
 
+  /// Store \p Target absolute adddress to \p RegName
+  virtual std::vector<MCInst> getAddress(const MCSymbol *Target, MCContext *Ctx,
```

Comment:
I think that getAddress might be a bit confusing, as it is not immediately obvious that it is asking the backend to generate a code sequence that is going to load an address into a register. Perhaps rename it to "materializeAddress"  or "create<something>"?

Reply:
Github reformatted my text. My suggestion was to use a name such as "createSomething", or "materializeAddress", whichever you prefer.